README.rst

```diff
@@ -26,26 +26,44 @@ Additional features:
                        an instance of your model, you can do so by adding a
                        dict to your model, which contains said additional data.
                        After creating that dict, use it when instantiating the
-                       HistoryLogging field.
+                       HistoryLogging field::
+
                             additional_data = {'changed_from':'djadmin'}
                             history = HistoryLogging(additional_data='additional_data')
+
    - Exclude fields -
-                      if you wish to not track some fields, all you need to do
+                      if you wish not to track some fields, all you need to do
                       is add a list to your model which contains the fields you
                       do not wish to track in a string format and use that list
-                      when instantiating the HistoryLogging field.
+                      when instantiating the HistoryLogging field::
+
                            exclude_fields = ['last_modified'] # as it would always appear to have been updated
                            history = HistoryLogging(exclude_fields='exclude_fields')
+
    - Ignore changes by user -
-                      if you wish to not track changes made by a specific user,
+                      if you wish not to track changes made by a specific user,
                       such as a user specially set up for smoke tests, you can declare
                       an additional field called however you like and pass it on
                       to the HistoryLogging object. This field must be a dictionary
                       that contains the keys 'user_ids' and 'user_names', both values
-                      for these keys must be lists containing the appropriate information.
+                      for these keys must be lists containing the appropriate information::
+
                            ignore_history_for_users = {'user_ids': [1010101], 'user_names': ['ignore_user']}
                            history = HistoryLogging(ignore_history_for_users='ignore_history_for_users')
 
+   - Interested related fields -
+                       *(added in version 1.1.0)*
+                       
+                       if you wish to add history for the objects related to a model
+                       when the model changes, you can do so by declaring a list with the names of
+                       the related fields names. This is applicable to 1-to-1, 1-to-many and
+                       many-to-many fields::
+
+                          poll = ForeignKey('Poll')
+                          ...
+                          interested_related_fields = ['poll']
```

Comment:
My gripe is that "interested" is not a word one would use on an object, an object can be OF interest, but it cannot be interested in something.